source/_integrations/calendar.markdown

```diff
@@ -130,7 +130,8 @@ automation:
         value_template: "{{ 'Front Lights' in trigger.calendar_event.summary }}"
     action:
       - if:
-          - "{{ trigger.event == 'start' }}"
```

Comment:
This is valid short-hand notation for a template condition

Reply:
Oh, hmm, well then I'm not sure why the Automations editor didn't treat it as valid syntax until I made this change - action wouldn't show up in the UI and wouldn't save. On the latest HA version. Will have another look.

Reply:
Not all valid YAML can be correctly represented in the UI. Try running it and see if it works 👍

Reply:
Ok, I had a chance to retry this - it does indeed save and work. Not sure why it wasn't saving before.

Whilst it would be best if the UI could represent this YAML, would it not be a better user experience if the sample code did indeed render in the UI? 
I'll rename the PR as it's not a 'fix', but as a user playing with this, I'd personally find YAML that renders in the UI as:
![image](https://user-images.githubusercontent.com/580744/220135428-ccbd7add-a2ab-41ec-8843-3724ad5297c6.png)
easier to use than YAML that renders as:
![image](https://user-images.githubusercontent.com/580744/220135475-35ee7feb-4aa1-472e-ad8f-406122bb618a.png)

Reply:
The current example is valid, if it results in a non-working UI, it should be reported in the issue tracker of the frontend repository.

Not all features available in YAML are available in the UI, however, in those cases a YAML block/editor should be shown in the UI.